features/steps/steps.ls

```diff
@@ -24,12 +24,9 @@ module.exports = ->
 
 
   @Given /^an instance of this service$/, (done) ->
-    port-reservation
-      ..get-port N (@service-port) ~>
-        @exocom.register-service name: 'tweets', port: @service-port
-        @process = new ExoService service-name: 'tweets', exocom-port: @exocom.pull-socket-port, exorelay-port: @service-port
-          ..listen!
-          ..on 'online', -> done!
+    @process = new ExoService exocom-host: 'localhost', service-name: 'tweets', exocom-port: @exocom-port
+      ..listen!
```

Comment:
We should probably change this to `connect` as well, to match Exorelay. I'll put up a PR for that.